src/project/cli-cmds/cmd-runner.ts

```diff
@@ -1,6 +1,5 @@
 import { BallerinaProject } from "../../core/extended-language-client";
-import { getCLIOutputChannel } from "./output";
-import { spawn, spawnSync } from "child_process";
+import * as  vscode from 'vscode';
```

Comment:
We can avoid the `*` import and use `Terminal` and `window.createTerminal()` where necessary.
```suggestion
import { Terminal, window } from "vscode";
```